docs/pages/deploy/deploy-button.zh-TW.mdx

```diff
@@ -8,44 +8,44 @@ import { Callout } from 'nextra-theme-docs'
 
 # 一鍵部署
 
-使用我們的一鍵部署按鈕，只需點選一下，就可以將服務部署到 Zeabur。按鈕可以在任何 Markdown 檔案或 HTML 檔案中使用。它可以將任何 [Zeabur 模板](https://zeabur.com/templates) 轉換為自動在 Zeabur 部署的專案。
+使用我們的一鍵部署按鈕，只需點選一下，就可以將服務部署到 Zeabur。按鈕可以在任何 Markdown 檔案或 HTML 檔案中使用。它可以將任何 [Zeabur 模板](https://zeabur.com/templates)轉換為自動在 Zeabur 部署的專案。
 
 ![button-demo](/deploy/button/button-demo.png)
 
 ## 按鈕的作用
 
-部署按鈕是一個簡單的 HTML 程式碼片段。您可以複製程式碼片段並貼到任何 Markdown 檔案或 HTML 檔案中。每個按鈕都將重定向使用者到所需的模板。模板可以是單個服務或由模板作者決定的一組服務。
+部署按鈕是一個簡單的 HTML 程式碼片段。你可以複製程式碼片段並貼到任何 Markdown 檔案或 HTML 檔案中。每個按鈕都將重新導向使用者到所需的模板。模板可以是單個服務或由模板作者決定的一組服務。
 
-按鈕也可以以作者的推薦碼結尾，每個使用推薦碼註冊並升級到付費方案的新使用者，作者將獲得 5 美元的積分。
+按鈕也可以以作者的推薦碼結尾，每個使用推薦碼註冊並升級到付費方案的新使用者，作者將獲得 5 美元的額度。
 
 ## 複製按鈕
 
-每個按鈕都是與模板相關的，只有模板作者才能產生按鈕。
+每個按鈕都與模板關聯，只有模板作者才能產生按鈕。
 
-想要複製按鈕，您需要在 [Zeabur Dashboard](https://dash.zeabur.com/account) 中的帳號頁面，選擇 **模板** 選項，您可以看到您建立的模板列表。
+想要複製按鈕，您需要在 [Zeabur 控制台](https://dash.zeabur.com/account)中的帳號頁面，選擇 **模板** 選項，您可以看到您建立的模板列表。
 
 ![created-templates](/deploy/button/created-templates.png)
 
 選擇所需的模板，然後點一下共享按鈕。將彈出一個視窗，其中包含按鈕程式碼。
 
 ![popup](/deploy/button/popup.png)
 
-如果您選擇啟用推薦碼，按鈕程式碼將附加推薦碼。推薦碼是您在 Zeabur 的使用者名稱。
+如果你選擇啟用推薦碼，按鈕程式碼將附加推薦碼。推薦碼是你的 Zeabur 使用者名稱。
 
 ## 使用場景
 
 這是將您的專案分享給世界的最佳方式。每個人都可以輕鬆地一鍵部署您的專案。
 
 比如，您可以將按鈕貼到以下位置：
 
-- GiHub 的 README 檔案
+- GitHub 的 `README.md` 檔案
```

Comment:
`README` is a term, and it can be written in a lot of formats. For example, `rst` and plain text.